lib/main.dart

```diff
@@ -76,19 +79,20 @@ class _RandomWordsState extends State<RandomWords> {
   }
 
   Widget _buildRow(WordPair pair) {
-    final alreadySaved = _saved.contains(pair);
+    final saved = _saved.contains(pair);
+
     return ListTile(
       title: Text(
         pair.asPascalCase,
         style: _biggerFont,
       ),
       trailing: Icon(
-        alreadySaved ? Icons.favorite : Icons.favorite_border,
-        color: alreadySaved ? Colors.orange : null,
+        saved ? Icons.favorite : null,
```

Comment:
So now, if the user didn't save some startup name, there will be just an empty space, won't it? 
How will the user become aware of the possibility of saving some startup name?

Reply:
https://github.com/Ari100telll/FlutterL1/pull/1#discussion_r716989073